rocAL/rocAL/include/pipeline/image.h

```diff
@@ -162,6 +167,7 @@ struct Image
 
     int create(vx_context context);
     void update_image_roi(const std::vector<uint32_t> &width, const std::vector<uint32_t> &height);
+    void update_image_orig_roi(const std::vector<uint32_t> &width, const std::vector<uint32_t> &height);
```

Comment:
Please change the parameter names to original_width and original_height
update_image_orig_roi(const std::vector<uint32_t> &original_width, const std::vector<uint32_t> &original_height);

Reply:
Resolved in the latest commit